### DISCUSSION TITLE
Private persistent peers visible in addr book

### DISCUSSION THREAD

**Discussion by skosito** [tendermint/tendermint#8429] [Q&A]
Hey,

I have a full node `A` where I set private peers in config toml like this (node id here is validator node):
```
private_peer_ids = "735......90d"
```

From my understanding, this one should not be propagated to other peers of this node. However, when I init new full node `B` and set its persistent peer to `A`, I still see 735.....90d in address book json on node `B`, more importantly its IP, which I wanted to prevent from happening. Am I doing something wrong here? Thanks

**Comment by creachadair** [comment]
Did the second node start up from (a copy of) the same configuration as the first, by chance?

**Reply by skosito** [reply]
@creachadair could you please clarify that a bit more, what do you mean by the same configuration?
